### PR TITLE
chore: markdownlint

### PR DESCRIPTION
**Pull request by rootulp**
Closes https://github.com/celestiaorg/CIPs/issues/93

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,11 +28,18 @@ Read [CIP-1](https://github.com/celestiaorg/CIPs/blob/main/cips/cip-1.md) for in
...
 | [17](/cips/cip-17.md) | Lemongrass Hardfork | Evan Forbes (@evan-forbes) |
 | [18](/cips/cip-18.md) | Standardised Gas and Pricing Estimation Interface | Callum Waters (@cmwaters) |
 
+## Contributing
+
+Files in this repo must conform to [markdownlint](https://github.com/DavidAnson/markdownlint). Install [markdownlint](https://github.com/DavidAnson/markdownlint) and then run:
+
+```shell
+markdownlint --config .markdownlint.yaml '**/*.md'
+```
```
Comment (jcstein) [inline README.md:45]:
```suggestion
```

```

I think mdlint requires newline at end of file?

File: cips/cip-template.md [hunk 2]
```diff
@@ -14,13 +14,13 @@ requires: CIP number(s). Only required when you reference an CIP in the `Specifi
 **READ CIP-1 BEFORE USING THIS TEMPLATE!**
 This is the suggested template for new CIPs. After you have filled in the requisite fields, please delete these comments. Note that an CIP number will be assigned by an editor. When opening a pull request to submit your CIP, please use an abbreviated title in the filename, `cip-draft_title_abbrev.md`. The title should be 44 characters or less. It should not repeat the CIP number in title, irrespective of the category.
 
-**TODO: Remove the note before submitting**
```
Comment (jcstein) [inline cips/cip-template.md:17]: [not blocking] this is the only thing that caught my eye @rootulp. should these remain bold? or a rule be ignored?

**Review by jcstein — APPROVED** [review]
left one non-blocking comment

File: cips/cip-template.md (hunk 2 above)
Comment (jcstein) [reply cips/cip-template.md:17]: yep it's MD036 https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md036---emphasis-used-instead-of-a-header

File: cips/cip-template.md [hunk 3]
```diff
@@ -14,13 +14,13 @@ requires: CIP number(s). Only required when you reference an CIP in the `Specifi
...
+TODO: Remove the note before submitting
```
Comment (jcstein) [inline cips/cip-template.md:17]:
```suggestion
> **TODO: Remove the note before submitting**
```

maybe this could work? or, just remove bold

**Review by YazzyYaz — APPROVED** [review]

File: cips/cip-template.md (hunk 2 above)
Comment (rootulp) [reply cips/cip-template.md:17]: Sync'ed with Josh offline and we agreed to remove the bold and keep the rule on.

**Pull request merged**